modules/telco-core-ref-eng-usecase-model.adoc

```diff
@@ -9,8 +9,13 @@
 The following engineering considerations are relevant for the telco core common use model.
 
 Worker nodes::
-* Worker nodes should run on Intel 3rd Generation Xeon (IceLake) processors or newer.
+* Worker nodes should run on either of the following CPUs:
+
+** Intel 3rd Generation Xeon (Ice Lake) 2.20 GHz CPUs or newer
+** AMD EPYC Zen 4 CPUs (Genoa, Bergamo, or newer)
 +
+The AMD EPYC Zen 4 CPUs (Genoa, Bergamo or newer) are fully supported. However, the power consumption evaluations are ongoing. It is recommended to evaluate features, such as per pod power management, to determine any potential impact.
```

Comment:
I think you are missing the [NOTE] designator here.

Reply:
I intentionally did not use a note here as there is another note next to this para. As per the the style guide, it is not recommended to stuff two notes one after the other. @MarSik